packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/Toolbar.tsx

```diff
@@ -68,7 +69,23 @@ const SelectWrapper = styled(Box)`
   }
 `;
 
-const ToolbarButton = ({ icon, name, label, isActive, disabled, handleClick }) => {
+interface ToolbarButtonProps {
+  icon: React.ComponentType;
+  name: string;
+  label: MessageDescriptor;
+  isActive: boolean;
+  disabled: boolean;
+  handleClick: () => void;
+}
```

Comment:
I would extend this as much as you can from `ToggleItemProps` from the radix package, this will help understand changes from updating packages.

Reply:
I understand the idea but I'm not sure how to apply it here:

- icon: specific to us, not passed to radix
- name: maps directly to radix but it's just a string
- label: specific to us as we're using react-intl. And I find it more convenient to call formatMessage inside the component than outside
- isActive: radix doesn't include data attributes in the types
- disabled: I want it mandatory, radix makes it optional
- handleClick: technically would be Radix's onMouseDown type, and it's optional when I want it required. I also don't want to pass it the event